Kickstarter-iOS/Views/Controllers/DiscoveryNavigationHeaderViewController.swift

```diff
@@ -17,13 +17,19 @@ internal final class DiscoveryNavigationHeaderViewController: UIViewController {
   @IBOutlet fileprivate weak var bookmarkImageView: UIImageView!
   @IBOutlet fileprivate weak var bookmarkOutlineImageView: UIImageView!
   @IBOutlet fileprivate weak var dividerLabel: UILabel!
+  @IBOutlet fileprivate weak var exploreLabel: UILabel!
   @IBOutlet fileprivate weak var favoriteButton: UIButton!
   @IBOutlet fileprivate weak var favoriteContainerView: UIView!
   @IBOutlet fileprivate weak var primaryLabel: UILabel!
   @IBOutlet fileprivate weak var secondaryLabel: UILabel!
   @IBOutlet fileprivate weak var titleButton: UIButton!
-  @IBOutlet fileprivate weak var titleStackView: UIStackView!
+  @IBOutlet fileprivate weak var containerStackView: UIStackView!
   @IBOutlet fileprivate weak var outerStackViewTopConstraint: NSLayoutConstraint!
+  @IBOutlet fileprivate weak var titleStackView: UIStackView! {
+    didSet {
+      titleStackView.pinBackground()
```

Comment:
🤔 interesting!

I like the approach of using `didSet` to apply one-time configurations on init. It's kind've like using `lazy var` for `@IBOutlet`s

Reply:
Yes, I would use `lazy var` for a variable that won't necessarily be allocated. Why do you like the `lazy var` approach for IBOutlets?

Reply:
No I'm saying I like overriding the setter and `didSet` because it keeps configuration of the UI element in one place and at the point at which it's defined.

Reply:
Ahh I understand. Yes, I like this approach to, so we don't have configuration code floating around